From a322e01d33a5da5b2db69628a6e3659ef47309cc Mon Sep 17 00:00:00 2001
From: Ben Shillito <ben@51degrees.com>
Date: Mon, 11 Jan 2016 17:33:24 +0000
Subject: [PATCH 08/67] DOC: add Ben Shillito as the maintainer of 51d

you can put me down as the maintainer with this email.

This should be backported to 1.6.
(cherry picked from commit eb90e8fcb9ffbfce09fb8fa525b953682a16a46d)
---
 MAINTAINERS | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/MAINTAINERS b/MAINTAINERS
index 06fca15..ea0060a 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -58,3 +58,6 @@ DeviceAtlas device identification
 Maintainer: David Carlier <dcarlier@afilias.info>
 Files: src/da.c, include/*/da.h
 
+51Degrees device identification
+Maintainer: Ben Shillito <ben@51degrees.com>
+Files: src/51d.c
-- 
2.4.10

